lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -221,21 +237,28 @@ public function __call($method, $arguments)
                 $method = 'findOneBy';
                 break;
 
+            case (0 === strpos($method, 'countBy')):
```

Comment:
In this new method, the called exception trigger should be replaced since it is specific to only `findBy*()` methods (`ORMException::invalidFindByCall()`). Should I create a new more generic `ORMException::invalidMagicCall()`, and deprecate the current?